src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,64 @@
+package duke.command;
+
+import duke.Action;
+import duke.DukeException;
+import duke.Parser;
+import duke.Storage;
+import duke.Ui;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.TaskList;
+import duke.task.Todo;
+
+public class AddCommand extends Command {
+    private final String info;
+
+    /**
+     * Constructs an AddCommand instance using the given action and info.
+     *
+     * @param action The given action.
+     * @param info The given info.
+     */
+    public AddCommand(Action action, String info) {
+        super(action);
+        this.info = info;
+    }
+
+    @Override
```

Comment:
perhaps you could write a javadoc for this method?

Reply:
I agree, public methods should have javadoc comments associated with them as per the coding standard.